misc/toml-validator/src/main/java/io/ballerina/toml/validator/models/Type.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.toml.validator.models;
```

Comment:
Since this is a C2C specific validator implementation, shouldn't this go to a C2C repo?

The base validator framework can be moved to the toml-parser module itself.

Reply:
This is not a C2C specific implementation. We use C2C as a test case, other than that, all the source code is entirely independent and can be used with any json schema and toml file.
That means we'll move everything to toml parser module right?